### ISSUE TITLE
Post action after pulling an image

### ISSUE THREAD

**Issue by benoitf** [containers/podman-desktop#10164]
### UX Description

Today, after pulling an image, there is a 'Done' button

suggestion is to be able to have a button to run the image

related topics:
- https://github.com/podman-desktop/podman-desktop/issues/10163
- https://github.com/podman-desktop/podman-desktop/issues/9977

### Request type

UX analysis/suggestions for improvement

**Comment by ekidneyrh** [comment]
Reposting my comment from #9977 

I think we should have this change in the pulling an image workflow and then have that be the flow accessible from the 'Create container from image' button instead of having it uniquely be associated with this action. 


Current flow:

1. Images
![Image](https://github.com/user-attachments/assets/7b435816-a504-4883-abc0-4d1235acdff5)

2. Pull image
![Image](https://github.com/user-attachments/assets/92838e8a-eedc-4459-a8b3-6b41e427ed72)

3. Done
![Image](https://github.com/user-attachments/assets/9a8396ee-efc4-4fa0-bcfe-18bbfda5f4a3)

4. Return to Images & run image
![Image](https://github.com/user-attachments/assets/ba0a8dab-543b-4def-b3a9-65b5824ad37c)

5. Create container form
![Image](https://github.com/user-attachments/assets/352522c0-6df9-4979-bc97-37980216aba5)

So we would be looking at getting rid of step 4 and have step 3 direct to 5. 

We could solve this by having a 'run image' button on the pull image form along side a close button like so:

![Image](https://github.com/user-attachments/assets/cdd42d92-4c2c-4b33-8d02-c8d287113b32)

And then when you select run it bring you onto step 5.

**Comment by ekidneyrh** [comment]
Let me know if there is anything else needed from this - this is what I presume this is for but I could be mistaken :D

**Comment by deboer-tim** [comment]
Not directly related to this, but do we have a standard for form bottom buttons? I thought we had recent designs that had the buttons aligned right, but these are full width.

**Comment by ekidneyrh** [comment]
@deboer-tim yeah you're right! apologies for the mistake - I think I was referencing an older design. Here it is updated:

![Image](https://github.com/user-attachments/assets/ab86df3e-97d9-40bc-af56-fbd89e49ef47)

Let me know if anything else is needed

**Comment by ekidneyrh** [comment]
Closing as this seems complete on my end - feel free to reach out if anything else is needed
